src/components/annotations/attributes.js

```diff
@@ -19,7 +19,7 @@ module.exports = templatedArray('annotation', {
         valType: 'boolean',
         role: 'info',
         dflt: true,
-        editType: 'calcIfAutorange+arraydraw',
+        editType: 'calc+arraydraw',
```

Comment:
Not for now of course, but annotations (showing, hiding, dragging in GUI) are a great argument in favor of each trace and item tracking its own autorange contributions, so it doesn't take a full recalc to move one annotation.

Reply:
> so it doesn't take a full recalc to move one annotation.

Absolutely! I'll open an issue about this once this PR is merged.